internal/pkg/cli/svc_delete_test.go

```diff
@@ -341,17 +360,15 @@ func TestDeleteSvcOpts_Execute(t *testing.T) {
 					mocks.spinner.EXPECT().Start(fmt.Sprintf(fmtSvcDeleteStart, mockSvcName, mockEnvName)),
 					mocks.svcCFN.EXPECT().DeleteService(gomock.Any()).Return(nil),
 					mocks.spinner.EXPECT().Stop(log.Ssuccessf(fmtSvcDeleteComplete, mockSvcName, mockEnvName)),
-					// emptyECRRepos
-					mocks.ecr.EXPECT().ClearRepository(mockRepo).Return(nil),
 
-					// removeSvcFromApp
-					mocks.store.EXPECT().GetApplication(mockAppName).Return(mockApp, nil),
-					mocks.spinner.EXPECT().Start(fmt.Sprintf(fmtSvcDeleteResourcesStart, mockSvcName, mockAppName)),
-					mocks.appCFN.EXPECT().RemoveServiceFromApp(mockApp, mockSvcName).Return(nil),
-					mocks.spinner.EXPECT().Stop(log.Ssuccessf(fmtSvcDeleteResourcesComplete, mockSvcName, mockAppName)),
+					// It should **not** emptyECRRepos
+					mocks.ecr.EXPECT().ClearRepository(mockRepo).Return(nil).Times(0),
```

Comment:
nit: can we make the arguments `gomock.Any()` since we want to check that it's not called at all regardless of the inputs

Reply:
sure sure